.github/workflows/update-ubuntu-version.yml

```diff
@@ -0,0 +1,58 @@
+name: "update-ubuntu-version"
+on:
+  workflow_dispatch:
+  schedule:
+    # every Monday at around 3 am pacific/10 am UTC
+    - cron: "0 10 * * 1"
+env:
+  GOPROXY: https://proxy.golang.org
+  GO_VERSION: '1.20.3'
+permissions:
+  contents: read
+
+jobs:
+  bump-ubuntu-version:
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@8f4b7f84864484a7bf31766abe9204da3cbe65b3
+      - uses: actions/setup-go@4d34df0c2316fe8122ab82dc22947d607c0c91f9
+        with:
+          go-version: ${{env.GO_VERSION}}
+          cache-dependency-path: ./go.sum
+      - name: Bump Ubuntu version
+        id: bumpUbuntu
+        run: |
+          make update-ubuntu-version
+          c=$(git status --porcelain)
+          c="${c//$'\n'/'%0A'}"
+          c="${c//$'\r'/'%0D'}"
+          echo "changes<<EOF" >> $GITHUB_OUTPUT
+          echo "$c" >> $GITHUB_OUTPUT
+          echo "EOF" >> $GITHUB_OUTPUT
+      - name: Create PR
```

Comment:
Will need to add:
```
id: createPR
```

So `${{ steps.createPR.outputs.pull-request-number }}` can get the PR number

Reply:
Sorry about that. Took the code snippet from the `update-docker-version` and didn't realize this was missing. I think it is solved now.